app/graphql/types/locale_enum.rb

```diff
@@ -1,7 +1,7 @@
 module Types
   class LocaleEnum < Types::BaseEnum
-    FastGettext.human_available_locales.each do |description, locale|
-      value locale, description: description
+    FastGettext.default_available_locales.each do |locale|
+      value locale, description: locale
```

Comment:
Just for my own understanding, this property is defined here:
https://github.com/theforeman/foreman/blob/7f25f3941f2a96240841aeb0903c46c28da8b212/lib/foreman/gettext/support.rb#L45-L58

That means this is reading the same list of locales. However, it is removing the localized description which can be considered a breaking change since it is an external API.

Just thinking out loud here. We do load all locales in production. Should we have 3 methods:
* `human_available_locales` - always a list of tuples but may not be translated
* `human_available_locales!` - always a list of tuples and loads translations if needed (I don't see an immediate need for it now)
* `register_human_localenames` - does the actual loading of translations

Would that actually work?

Reply:
I am not sure what you mean by "here" but FastGettext.locale is defined here:

https://github.com/grosser/fast_gettext/blob/master/lib/fast_gettext/storage.rb#L105-L107

And FastGettext.default_available_locales is defined here:

https://github.com/grosser/fast_gettext/blob/master/lib/fast_gettext/storage.rb#L33-L57

Reply:
With "this property" I meant the old `human_available_locales` property. Looks like I commented on the wrong line.

Reply:
We are not breaking the API too much. LocaleEnum is only used in User model for user's locale. Writes should be not affected at all, reading changed from `cs_CZ = Česky` to `cs_CZ = cs_CZ`. I think it is acceptable in the context of shaving off 2 seconds from every boot.

Anyway, is GraphQL considered a stable API? Aren't we just wasting our time here? I asked @timogoebel three months ago to double-check because they are one (and maybe the only one) GraphQL users at the moment. I wish I had the answer.

Reply:
Aren't we having `cs_CZ = cs_CZ` now?
But I think it's ok, GraphQL is still not stable API and this is definitely not something someone should depend on...

Reply:
Yeah sorry it is a bit worse than that.